app/models/beta_pages.rb

```diff
@@ -1,8 +1,6 @@
 class BetaPages
   def self.all
     [
-      'agent/clusters',
```

Comment:
Just an FYI, this page/path doesn't/no longer seems to exist and so I'm removing it as part of this PR.